recipes/ogre/1.x/conanfile.py

```diff
@@ -0,0 +1,285 @@
+        "OGRE_INSTALL_VSPROPS": False,
+        "OGRE_NODELESS_POSITIONING": True,
+        "OGRE_PROFILING_REMOTERY_PATH": "",
+        "OGRE_RESOURCEMANAGER_STRICT": 2,
+        "OGRE_STATIC": True,
+        "OGRE_SET_DOUBLE": False,
+        "OpenEXR_Half_LIBRARY": "OpenEXR_Half_LIBRARY-NOTFOUND",
+        "OpenEXR_Half_LIBRARY_DEBUG": "OpenEXR_Half_LIBRARY_DEBUG-NOTFOUND",
+        "OpenEXR_INCLUDE_DIR": "OpenEXR_INCLUDE_DIR-NOTFOUND",
+        "OpenEXR_Iex_LIBRARY": "OpenEXR_Iex_LIBRARY-NOTFOUND",
+        "OpenEXR_Iex_LIBRARY_DEBUG": "OpenEXR_Iex_LIBRARY_DEBUG-NOTFOUND",
+        "OpenEXR_IlmImf_LIBRARY": "OpenEXR_IlmImf_LIBRARY-NOTFOUND",
+        "OpenEXR_IlmImf_LIBRARY_DEBUG": "OpenEXR_IlmImf_LIBRARY_DEBUG-NOTFOUND",
+        "OpenEXR_IlmThread_LIBRARY": "OpenEXR_IlmThread_LIBRARY-NOTFOUND",
+        "OpenEXR_IlmThread_LIBRARY_DEBUG": "OpenEXR_IlmThread_LIBRARY_DEBUG-NOTFOUND",
+        "Qt5_DIR": "Qt5_DIR-NOTFOUND",
+        "SDL2_DIR": "${ogre-build-dir}/Dependencies/cmake",
+        "pugixml_DIR": "${ogre-build-dir}/Dependencies/lib/cmake/pugixml"
+    }
+    exports_sources = "CMakeLists.txt", "patches/**"
+    _cmake = None
+
+    def requirements(self):
+        self.requires("freeimage/3.18.0")
+        self.requires("boost/1.75.0")
+        self.requires("freetype/2.11.1")
+        self.requires("openexr/2.5.7")
+        self.requires("poco/1.11.2")
+        self.requires("tbb/2020.3")
+        self.requires("zlib/1.2.12")
+        self.requires("zziplib/0.13.71")
+
+    def validate(self):
+        """
+         OGRE 1.x is very old and will not work with latest gcc, clang and msvc compielrs
+         TODO: determine incompatible msvc compilers
+        """
+        if self.settings.compiler == "gcc" and self.settings.compiler.version == 11:
+            raise ConanInvalidConfiguration("OGRE 1.x not supported with gcc 11")
+        if self.settings.compiler == "clang" and self.settings.compiler.version == 13:
+            raise ConanInvalidConfiguration("OGRE 1.x not supported with clang 13")
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        self.options.OGRE_STATIC = not self.options.shared
+
+    def _configure_cmake(self):
+        if self._cmake is not None:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["OGRE_CONFIG_DOUBLE"] = False
+        self._cmake.definitions["OGRE_CONFIG_NODE_INHERIT_TRANSFORM"] = False
+        self._cmake.configure(source_folder=self._source_subfolder)
+        return self._cmake
+
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        conan.tools.files.rename(
+                self,f"ogre-{self.version}", self._source_subfolder
+            )
```

Comment:
```suggestion
        tools.get(**self.conan_data["sources"][self.version], destination=self._source_subfolder, keep_root=True)
```
tools.get is able to do all steps ;)